<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LivePlayerDialog</class>
 <widget class="QDialog" name="LivePlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LivePlayerDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>515</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QLabel" name="m_showLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>770</width>
      <height>515</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="m_curtTimeLabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>532</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="m_progressSlider">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>532</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="m_totalTimeLabel">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>532</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="m_resumePB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>562</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #000000;</string>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pausePB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>562</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #000000;</string>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>230</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QPlainTextEdit" name="m_plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>495</y>
      <width>230</width>
      <height>105</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border-top: 1px solid #000000;
border-left: 1px solid #000000;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_sendMsgPB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>565</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #000000;</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="m_scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>230</width>
      <height>395</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>230</width>
       <height>395</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>230</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 1px solid #000000;
border-bottom: 1px solid #000000;</string>
    </property>
    <widget class="QLabel" name="m_liveRoomOwnerAvatarLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top: 0px solid #000000;
border-bottom: 0px solid #000000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/res.qrc">:/img/avatar.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="m_liveRoomOwnerNameLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>15</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top: 0px solid #000000;
border-bottom: 0px solid #000000;</string>
     </property>
     <property name="text">
      <string>花未眠</string>
     </property>
    </widget>
    <widget class="QLabel" name="m_liveRoomViewerCountLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top: 0px solid #000000;
border-bottom: 0px solid #000000;</string>
     </property>
     <property name="text">
      <string>人数: 1000人</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1px solid #000000;</string>
    </property>
    <property name="text">
     <string>  消息</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
